test/src/main/scala/scalarules/test/resource_jars/BUILD

```diff
@@ -1,7 +1,19 @@
+load("@rules_java//java:defs.bzl", "java_library")
 load("//scala:scala.bzl", "scala_test")
 
+java_library(
+    name = "jarWithResources",
+    resources = [
+        "//test/src/main/resources/scalarules/test:byes",
+        "//test/src/main/resources/scalarules/test:hellos",
+        "//test/src/main/resources/scalarules/test:more-byes",
+        "//test/src/main/resources/scalarules/test:more-hellos",
+    ],
+    visibility = ["//visibility:public"],
+)
+
 scala_test(
     name = "resource_jars",
     srcs = ["TestResourceJars.scala"],
-    resource_jars = ["//test/src/main/resources/scalarules/test:hellos-and-byes.jar"],
+    resource_jars = [":jarWithResources"],
```

Comment:
please refer explicitly to `.jar` output - it will be easier to follow test meaning